hackathon_site/hardware/serializers.py

```diff
@@ -39,6 +39,25 @@ def get_unique_hardware_count(obj: Category) -> int:
         return obj.hardware_set.annotate(Count("id", distinct=True)).count()
 
 
+class IncidentsSerializer(serializers.ModelSerializer):
```

Comment:
Will need a test for this serializer, in `hardware/tests.py`. Since there's nothing special going on in the serializer, you should be good with just one test method in the class where you hard-code the expected JSON response. Since time fields are annoying to fix in tests, take a look at [this](https://github.com/ieeeuoft/hackathon-template/blob/develop/hackathon_site/hardware/tests.py#L204) order serializer test for how to do it.